packages/core-components/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -27,7 +27,8 @@ import { MicDrop } from './MicDrop';
 interface IErrorPageProps {
   status: string;
   statusMessage: string;
-  additionalInfo?: string;
+  additionalInfo?: string | JSX.Element;
```

Comment:
Is there any specific reason not to use ReactElement? Just curiosity.

Reply:
Honestly just me and that I always get to the question "When to use JSX.Element vs ReactNode vs ReactElement?" 😆 Can you let me know about the recommendations please? I wonder if even `ReactNode` is preferred here as it seems like it can be `a ReactElement, a ReactFragment, a string, a number or an array of ReactNodes, or null, or undefined, or a boolean`

Reply:
I think - and I say this as a not-at-all-react-expert-really - that it's fairly safe to say that

- For inputs to functions, things that will turn out to be rendered by the function - use `React.ReactNode` because that encompasses all things that React can "eat"
- For outputs from functions (components mostly), things that are already rendered - use `JSX.Element` or even `JSX.Element | null` if necessary (that last one is a bit of a quirk; null isn't part of the possible jsx outputs but components still want to return null sometimes)

Reply:
I don't think I ever used `ReactElement`; only `ReactNode` which also is the expected type of `children` by default

Reply:
I think ReactElement is an instance of a React component and all React Elements are `JSX.Elements` which contains type (component type), props (props passed to component), key ( key of the component). I use ReactElement when I expect to receive a JSX.Element, but this might not be correct, so before suggesting I asked to see if it would make sense 🙂

Reply:
Resolving the thread, it was only curiosity 🙌🏻

Reply:
For clarity, `ReactElement` defines `type` and `props` on an `interface`.

 A `ReactNode` is of type` ReactChild | ReactElement | ReactFragment | ReactPortal | null | undefined`, basically everything you can render.
 
 And `JSX.Element` is  `ReactElement` with the `type` and `props` set as `any`.

I'd say if you don't care about the thing you're taking and don't want to check the props on the component your taking in, use `JSX.Element` however if you do, take `ReactElement<type,props>`.

It's used quite a lot for some of the `composability` and `extensions` work, so it does have it's place :)

Reply:
But I still think that for most "everyday" use cases, like in this instance when you accept some additional info that you'll only plop in your rendered output somewhere, you do want to accept strings and everything else. ReactNode is perfect for this.

Reply:
I think this discussion was sooo intresting, thanks for sharing your knowledge and thanks @camilaibs for brining up the question in the first place!! Based on the information shared above, I think it makes sense to update the type to `ReactNode `. Will do!